src/main/java/com/jalasoft/jfc/model/VideoConverter.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not disclose such Confidential
+ *  Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.model;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+/**
+ *
+ *  VideoConverter Class is used for convert videos.
+ *
+ * @version 1.0 13 Dic 2019
+ *
+ * @author Juan Martinez
+ */
+public class VideoConverter {
+    /**
+     * This method convert a video format to another format.
+     * @param videoParam is an object of videoParam class.
+     * @return int cor confirm the video conversion.
```

Comment:
What is "int cor?"

Reply:
It was a miss spelling, it variable for returning the result